README.md

```diff
@@ -170,8 +171,8 @@ $msg = $this->get('m6_web_amqp.consumer.myconsumer')->getMessage();
 The consumer does not wait for a message : getMessage will return null immediately if no message is available or return a AMQPEnvelope object if a message can be consumed.
 The "flags" argument of getMessage accepts AMQP_AUTOACK (auto acknowledge by default) or AMQP_NOPARAM (manual acknowledge).
 
-To manually acknowledge a message, use the consumer's ackMessage/nackMessage methods with a delivery_tag argument's value from the AMQPEnvelope object. 
-If you choose to not acknowledge the message, the second parameter of nackMessage accepts AMQP_REQUEUE to requeue the message or AMQP_NOPARAM to forget it. 
+To manually acknowledge a message, use the consumer's ackMessage/nackMessage methods with a delivery_tag argument's value from the AMQPEnvelope object.
```

Comment:
Err, you're right, [it's `2 spaces` at the end of the line](http://markdown-guide.readthedocs.io/en/latest/basics.html#line-return) not just `1`, my bad 🤕 
So this modification won't change anything 👍